.github/workflows/documentation.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Run pre-requisites for validation
         run: |
           make -C Documentation copy-api # necessary for check-build.sh
-      - uses: docker://cilium/docs-builder:2023-02-04@sha256:67c383332f63708fc40f3d12b0ffff8ec94a9a83d1b819dbe23b226ef076e6f2
+      - uses: docker://qmonnet/docs-builder:2023-02-24
```

Comment:
cilium/docs-builder:2023-02-24 is on the way. I think you should be able to retrieve the sha from here once it's built? https://hub.docker.com/repository/registry-1.docker.io/cilium/docs-builder/builds/a4d7572f-ce1b-414b-9243-2e28d2f64815